test/ZNSRootRegistrar.test.ts

```diff
@@ -154,6 +161,123 @@ describe("ZNSRootRegistrar", () => {
     );
   });
 
+  it("Throws if env variable is invalid", async () => {
+    try {
+      const config = await getConfig(
+        deployer,
+        zeroVault,
+        [deployer.address, governor.address],
+        [deployer.address, admin.address],
+      );
+
+      validate(config, "other");
```

Comment:
see this: https://github.com/zer0-os/zNS/pull/73/files#r1395017387

Please, set the ENV vars in the proper way here so that the validation test actually tests a real-life scenario. right now a discrepancy is possible where ENV_LEVEL can be set to a different value compared to this argument hardcoded in the test.